### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by PeterPedegru** [entropyxyz/community#126] [Get Test Funds] [answered]
### Address

5F79tgYteELka1pEEw4K94YS6fJUTZKwkGLsLP81RoN6GbCB

**Comment by johnnymatthews** [comment, marked as answer]
Sent in block [584,393](https://polkadot.js.org/apps/#/explorer/query/0xa72b70a389176ddc3172cb00cacb3eed03b4f2dd30cfd7fed4c02d68f4317799)
